lib/bgs/dependents.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+module BGS
+  class Dependents < Service
+    include Helpers::Formatters
+
+    def initialize(proc_id:, payload:, user:)
+      @proc_id = proc_id
+      @payload = payload
```

Comment:
What form is `payload` supposed to take here? Maybe it could be a model that we could validate before proceeding? Because it looks like this service expects this object to contain a lot of things.

Reply:
`payload` is a massive hash. The form allows users to add unlimited dependents in some workflows.